### ISSUE TITLE
Attribute values cannot be set on the creation of a resource.

### ISSUE THREAD

**Issue by barelyknown** [pixelhandler/ember-jsonapi-resources#13]
The `attributes` object is not set up until `init()` is called, but that happens after `create()`. So, there's an error if you try to set an attribute during create.

```javascript
import Resource from 'ember-jsonapi-resources/models/resource';
const Game = Resource.extend({
  type: 'games',
  name: attr()
});
Game.create({name: 'Foo'});
// Error: Property set failed: object in path "attributes" could not be found or was destroyed.
```

I'm not sure what the idiomatic way is to fix this, but the following works.

```javascript
Resource.reopenClass({
  create(...args) {
    let i = this._super();
    i.setProperties(...args);
    return i;
  }
});
```

**Comment by barelyknown** [comment]
My comment on https://github.com/pixelhandler/ember-jsonapi-resources/pull/12 gives some context to why I was setting the attribute via its `attr()` instead of setting the `attributes` object during `create()`.

**Comment by pixelhandler** [comment]
Should be fixed with PR #14
